lib/gems/pending/VMwareWebService/MiqVimInventory.rb

```diff
@@ -64,11 +65,17 @@ def initialize(server, username, password, cacheScope = nil)
         deleteProperty(:HostSystem, "config.dateTimeInfo")
       end
     else
-      if @v4 && cacheScope != :cache_scope_event_monitor
+      if @v4
         @propMap = dupProps(@propMap)
-        addProperty(:VirtualMachine, "runtime.memoryOverhead")
-        deleteProperty(:VirtualMachine, "config.hardware.numCoresPerSocket")
+
+        deleteProperty(:Datastore, "summary.maintenanceMode")
+
+        if cacheScope != :cache_scope_event_monitor
+          addProperty(:VirtualMachine, "runtime.memoryOverhead")
+          deleteProperty(:VirtualMachine, "config.hardware.numCoresPerSocket")
```

Comment:
@roliveri actually I don't fully understand this block, why is the `deleteProperty(:VirtualMachine, "config.hardware.numCoresPerSocket")` in `if cacheScope != :cache_scope_event_monitor` for `@v4` but not [here](https://github.com/agrare/manageiq-gems-pending/blob/7aab7da6d2624d0b984281c0b3b529d1233efc2d/lib/gems/pending/VMwareWebService/MiqVimInventory.rb#L54) in `@v2`?  I get not wanting to do the `addProperty` since that wouldn't be in the `EventMonitorPropMap` anyway but is that the only one that should be in the `if cacheScope != :cache_scope_event_monitor` conditional?